src/spec/helpers/functions_helper_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'rails_helper'
+
+RSpec.describe FunctionsHelper do
+  describe '#render_markdown' do
+    context 'normal text' do
+      it { expect(helper.render_markdown('text')).to match('<p>text</p>') }
+    end
+
+    context 'bold text' do
+      it { expect(helper.render_markdown('**bold**')).to match('<p><strong>bold</strong></p>') }
+    end
+
+    context 'italic text' do
+      it { expect(helper.render_markdown('*bold*')).to match('<p><em>bold</em></p>') }
+    end
+
+    context 'ordered list' do
+      it do
+        markdown_list = <<~ACTUAL
+          1. First
+          2. Second
+        ACTUAL
+
+        html_list = <<~EXPECTED
+          <ol>
+            <li>First</li>
+            <li>Second</li>
+          </ol>
+        EXPECTED
+
+        expect(helper.render_markdown(markdown_list)).to match(html_list)
+      end
+    end
+
+    context 'unordered list' do
+      it do
+        markdown_list = <<~ACTUAL
+          * First
+          * Second
+        ACTUAL
+
+        html_list = <<~EXPECTED
+          <ul>
+            <li>First</li>
+            <li>Second</li>
+          </ul>
+        EXPECTED
+
+        expect(helper.render_markdown(markdown_list)).to match(html_list)
+      end
+    end
+
+    context 'links' do
+      it do
+        markdown_link = '[RubyFunctions](https://runyfunctions.com)'
+        html_link = '<p><a href="https://runyfunctions.com">RubyFunctions</a></p>'
```

Comment:
should have 
https://developers.google.com/web/tools/lighthouse/audits/noopener
```suggestion
        html_link = '<p><a href="https://runyfunctions.com" rel="noopener">RubyFunctions</a></p>'
```